mqtt/mqtt.js

```diff
@@ -87,125 +85,81 @@ nats.subscribe('channel.*', function (msg) {
  */
 // AuthZ PUB
 aedes.authorizePublish = function (client, packet, callback) {
-    // Topics are in the form `mainflux/channels/<channel_id>/messages/senml-json`
-    var channel = packet.topic.split('/')[2];
-
-    /**
-     * Check if PUB is authorized
-     */
-    var options = {
-        url: config.auth_url + ':' + config.auth_port + '/channels/' + channel + '/access-grant',
-        method: 'GET',
-        headers: {
-            'Content-Type': 'application/json',
-            'Authorization': client.password
-        }
-    };
-
-    request(options, function (err, res) {
-        var error = null;
-        var msg = {};
-        if (res && (res.statusCode === 200)) {
+    // Topics are in the form `channels/<channel_id>/messages/senml-json`
+    var channel = packet.topic.split('/')[1];
+
+    thingsClient.CanAccess({
```

Comment:
You can safely omit `Client` suffix.

Reply:
I already have `things` here: `var things = grpc.load("../internal.proto").mainflux;`.